clickhouse/columns/enum.cpp

```diff
@@ -90,6 +90,19 @@ ColumnRef ColumnEnum<T>::Slice(size_t begin, size_t len) {
     return std::make_shared<ColumnEnum<T>>(type_, SliceVector(data_, begin, len));
 }
 
+template <typename T>
+void ColumnEnum<T>::Swap(Column& other) {
+    if (auto col = dynamic_cast<ColumnEnum<T>*>(&other)) {
+        data_.swap(col->data_);
+    }
+}
+
+template <typename T>
+ItemView ColumnEnum<T>::GetItem(size_t index) const {
+    const T value = data_[index];
+    return ItemView{type_->GetCode(), value};
```

Comment:
Just looking at this isolated piece, it feels more safe performance wise to avoid potential creation of unnecessary local copy here. I.e, suggesting changing to this: 
```c++
    return ItemView{type_->GetCode(), data_[index]};
```

Reply:
well, that would most likely not make any difference, but for sake of simplicity and brievety, fixed